### ISSUE TITLE
alternate method for encoding urls

### ISSUE THREAD

**Issue by Bhupesh-V** [VonHeikemen/dotfiles#1]
Hello Heiker :wave: 
Came to know about this repo from your DEV.to comment

I am curious Any reason why `jq` is preferred here ?
https://github.com/VonHeikemen/dotfiles/blob/3e2607dd0a827f71122bbb51a5e04f1cfa906322/my-configs/zsh/custom-functions.zsh#L86-L96

I just researched & found vanilla bash can do it too.

```bash
read -p "Search Query : " search_term
#search_term='bhupesh varshney is a good boi give him monies'

# replace spaces with +
urlencode="${search_term// /+}"

# Replace firefox with either "chromium-browser" or command to open your fav browser
firefox --new-tab "lite.duckduckgo.com/lite/?q=${urlencode}"
#echo -e "$urlencode"
```

Anyways, Thanks for this idea :sparkles:

**Comment by VonHeikemen** [comment]
It's not just about replacing whitespace. Consider the string `this & that`, if I'm not wrong that will get you `this+&+that`. The search will only reflect the string `this`.

There are of course more options. I got that mine from [this stackoverflow question](https://stackoverflow.com/questions/296536/how-to-urlencode-data-for-curl-command). I chose jq because I already have it installed and it's a short command.
